### PR TITLE
DEATH SQUAD gear tweaks & leader ID fix

### PR DESCRIPTION
**Pull request by SonixApache**
+ mateba now starts holstered in holster on uniform
+ pulse rifle spawns on belt instead of in-hand
+ esword spawns in boot knife holster
+ .357 speedloader instead of ammo box

🆑 
 - tweak: Certain striketeams that may or may not exist now come with gun and sword holsters

### CODE REVIEW

**Comment by Irradiation** [comment]
>PR called nuke ops gear tweak
>Doesn't touch Nuke Ops gear

**Comment by tuypo1** [comment]
This is for the deathsquad though, did you have a bit of confusion uploading this alongside your nuke op map changes?

**Comment by SonixApache** [comment]
don't sleep and code, kids

**Comment by Kharkov72** [comment]
[tested] when

**Comment by ComradeDispenser** [comment]
E swords fit in boot holsters? Damn, the more you know

**Comment by SonixApache** [comment]
this one was tested extensively, just fucked up the title initially

**Comment by DeityLink** [comment]
what the fuck is that line for https://github.com/vgstation-coders/vgstation13/pull/23561/files#diff-43b77aed78899890ef7ca45e2bbeaca7R125
@SonixApache Fix Major Creed spawning with the wrong ID, which makes him unable to board the Seraph.

It should have

```
W.access = get_centcom_access("Creed Commander")
W.icon_state = "creed"
```

**Comment by SonixApache** [comment]
Creed is the leader, right?

File: code/game/striketeams/nanotrasen_deathsquad.dm [hunk 1]
```diff
@@ -122,10 +122,15 @@
...
-	W.icon_state = "centcom"
-	W.access = get_centcom_access("Death Commando")
-	W.icon_state = "deathsquad"
-	W.assignment = "Death Commando"
+	if(leader)
+		W.access = get_centcom_access("Creed Commander")
+		W.icon_state = "creed"
+		W.assignment = "Creed Commander"
+	else
+		W.icon_state = "centcom"
```
Comment (DeityLink) [inline code/game/striketeams/nanotrasen_deathsquad.dm]: that line is still useless, you're changing it again two lines bellow.

File: code/game/striketeams/nanotrasen_deathsquad.dm [hunk 2]
```diff
@@ -121,10 +122,14 @@
...
 	var/obj/item/weapon/card/id/W = new(src)
 	W.name = "[real_name]'s ID Card"
-	W.icon_state = "centcom"
-	W.access = get_centcom_access("Death Commando")
-	W.icon_state = "deathsquad"
-	W.assignment = "Death Commando"
+	if(leader)
+		W.access = get_centcom_access("Creed Commander")
+		W.icon_state = "creed"
+		W.assignment = "Creed Commander"
```
Comment (Kurfursten) [inline code/game/striketeams/nanotrasen_deathsquad.dm]: Is this supposed to be "Commander Creed"?

Comment (SonixApache) [reply code/game/striketeams/nanotrasen_deathsquad.dm]: nope

Comment (DeityLink) [reply code/game/striketeams/nanotrasen_deathsquad.dm]: It's only half of a WH40k reference. In my headcanon creed was more of a title. Could always be changed.

Comment (SonixApache) [reply code/game/striketeams/nanotrasen_deathsquad.dm]: tell me what to make it and i will

**Pull request merged**